ratom/views.py

```diff
@@ -1,3 +1,178 @@
-from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 
-# Create your views here.
+from django_elasticsearch_dsl_drf import constants, filter_backends
+from django_elasticsearch_dsl_drf.viewsets import DocumentViewSet
+from elasticsearch_dsl import DateHistogramFacet, RangeFacet, TermsFacet
+from ratom.permissions import IsOwnerOrReadOnly
+
+from .documents.message import MessageDocument
+from .models import User, Account, Message
+from .serializers import (
+    UserSerializer,
+    AccountSerializer,
+    MessageSerializer,
+    MessageDocumentSerializer,
+)
+
+
+@api_view(["GET"])
+@permission_classes([IsAuthenticated])
+def user_detail(request):
+    """
+    Show details of single user
+    """
+    try:
+        user_pk = request.user.pk
+        user = User.objects.get(pk=user_pk)
+    except User.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+
+    if request.method == "GET":
+        serialized_user = UserSerializer(user)
+        return Response(serialized_user.data)
+
+
+@api_view(["GET", "POST"])
+@permission_classes([IsAuthenticated])
+def account_list(request):
```

Comment:
I don't think we need to change, but I wonder what the class-based views would look like here: https://www.django-rest-framework.org/api-guide/generic-views/#concrete-view-classes

Reply:
I'm totally open to making the change, if we want to. Using function based views feels more easily customizable to me, but that's probably just a comfort-level thing.